### PR TITLE
Cross linking Learn tutorials

### PR DESCRIPTION
**Pull request by danielehc**
Here the first changes to the documentation to cross link the Learn tutorials.

I am not sure if we want to add links in other pages but, once https://github.com/hashicorp/learn/pull/3096 is merged we might want to change the first link in https://www.consul.io/docs/nia/terraform-modules#how-to-create-a-compatible-terraform-module to the new specific CTS tutorial or add an extra one.

Comments or changes are welcome.

### CODE REVIEW

File: website/content/docs/nia/index.mdx [hunk 1]
```diff
@@ -32,6 +32,15 @@ Consul-Terraform-Sync executes one or more automation tasks with the most recent
 
 **Terraform Provider** - A [Terraform provider](https://www.terraform.io/docs/providers/index.html) is responsible for understanding API interactions and exposing resources for an infrastructure type.
 
+## Getting Started With Network Infrastructure Automation
+
+The [Network Infrastructure Automation (NIA)](https://learn.hashicorp.com/collections/consul/network-infrastructure-automationutm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS)
```
Comment (findkim) [inline website/content/docs/nia/index.mdx:37]: I like the threading of Learn links here. Though this one doesn't seem to work and I get Page Not Found when clicking from the vercel preview app

image

Comment (findkim) [reply website/content/docs/nia/index.mdx:37]: Oh i think it's just missing a `?`

```suggestion
The [Network Infrastructure Automation (NIA)](https://learn.hashicorp.com/collections/consul/network-infrastructure-automation?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) 
```

File: website/content/docs/nia/architecture.mdx [hunk 2]
```diff
@@ -50,3 +50,9 @@ A driver encapsulates the resources required to communicate the updates to the
 network infrastructure. Terraform is the initial supported driver. It comes
 with a set of providers that [enables supporting](/docs/nia/terraform-modules) a wide variety of
 infrastructure applications.
+
+## Security Guidelines
+
+The [Secure Consul-Terraform-Sync for Production](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-secure?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS)
```
Comment (findkim) [inline website/content/docs/nia/architecture.mdx:56]: The secure learn guide would be great to include in the configuration portion on the Consul client token

File: website/content/docs/nia/installation/run.mdx [hunk 3]
```diff
@@ -27,3 +27,12 @@ description: >-
   ```shell-session
   $ curl localhost:8558/status/tasks
   ```
+
+## Other Run modes
```
Comment (findkim) [inline website/content/docs/nia/installation/run.mdx:31]: I'm a bit split on where this should be, here or within the [CLI modes](https://www.consul.io/docs/nia/cli/cli-overview#modes) page. My preference would be the latter. I do see value in threading the learn guides within the "Get Started" navigation section, so maybe both location?

Comment (danielehc) [reply website/content/docs/nia/installation/run.mdx:31]: I voluntarily didn't mess up with the CLI and API docs.

I am adding the intro link to the install page, so ideally they could find it as soon as they get interested in getting hands on and use the katacoda lab for testing.

Comment (findkim) [reply website/content/docs/nia/installation/run.mdx:31]: those are good places to add!

**Review by findkim — APPROVED** [review]

**Comment by findkim** [comment]
Not a necessary addition, but wanted to bring it up in case you thought it was worthwhile: NIA has a dedicated security page that captures the threat model and mentions ACLs there. Would you be interested in adding the Secure learn guide there too or would this be too much?

https://www.consul.io/docs/security/security-models/nia

**Comment by findkim** [comment]
The PR contains changes to only website pages and do not effect the failing integration test with envoy. Since it doesn't look like i can rerun that test, I'll go ahead and 👍this PR it strikes me as safe to merge.

**Comment by findkim** [comment]
Looks like i was able to re-run the test suite from GitHub even though I wasn't able to re-run the individual test on Circle. Everything is passing! ✅

**Pull request merged**